app/views/application/login.json.jbuilder

```diff
@@ -1,6 +1,5 @@
 json.partial! "users/user", user: @current_user
 json.partial! 'tags/tag', tags: @current_user.tags
```

Comment:
ha, I see! this is why you needed the `login`? I believe this was hacked up to serve some other product purposes! We don't really need this anymore, moreover, I believe there should be a route to fetch all the tags of the current user and the current information of the said user